dashboard/modules/job/job_manager.py

```diff
@@ -516,6 +516,12 @@ async def _monitor_job_internal(
                 is_alive = False
                 job_status = await self._job_info_client.get_status(job_id)
                 job_error_message = None
+                if job_status == JobStatus.FAILED:
```

Comment:
this is unrelated to this PR, but it makes the failed job submission as an "error" event. cc @architkulkarni